samples/src/source/screens/TaskScreenVM.bs

```diff
@@ -0,0 +1,48 @@
+import "pkg:/source/roku_modules/maestro/mx/BaseViewModel.brs"
+import "pkg:/source/roku_modules/maestro/mx/VMTransformFunctions.brs"
+'
+namespace screens
+  class TaskScreenVM extends mx.BaseViewModel
+
+    public focusedIndex = -1
```

Comment:
thisis not the code formatting of the project. please don't use this formatting.

Reply:
How? I just wanna add a screen for the task example 

Thanks
Allan